easybuild/easyconfigs/h/HEALPix/HEALPix-3.70-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'ConfigureMake'
```

Comment:
@jose-d Any reason for not using the custom easyblock for HEALPix which we have already?

That would simplify the easyconfig file quite a bit...

See https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-GCCcore-8.2.0.eb